src/Vm.sol

```diff
@@ -1571,6 +1577,9 @@ interface VmSafe {
     /// Unpauses collection of call traces.
     function resumeTracing() external view;
 
+    /// Utility cheatcode to set arbitrary storage for given target address.
+    function setArbitraryStorage(address target) external;
```

Comment:
Do we have more docs for this in the foundry book? From this docstring alone it's not clear when/why to use this function or what it does exactly

Reply:
yes, I an creating a book PR with details and examples fir each of them